civis/io/_scripts.py

```diff
@@ -0,0 +1,63 @@
+import logging
+
+from civis import APIClient
+from civis.futures import CivisFuture
+
+log = logging.getLogger(__name__)
+
+
+def run_template(id, arguments, JSONValue=False):
+    """Run a template and return the results.
+
+    Parameters
+    ----------
+    id : int
+        The template id to be run.
+    arguments : dict
+        Dictionary of arguments to be passed to the template.
+    JSONValue : bool
+        If True, will return the JSON output of the template.
+        If False, will return the file ids associated with the
+        output results.
+
+    Returns
+    -------
+    If JSONValue = False:
+      file_ids : dict
+        Dictionary of file ids with the keys being their output names.
+    If JSONValue = True:
+      json_output : civis.response.Response
+        JSON object containing the results of the template run.
+        Expects only a single JSON result. Will return nothing if
+        either there is no JSON result or there is more than 1 JSON result.
+
+    Examples
+    --------
+    >>> # Run template to return file_ids
+    >>> run_template(my_template_id, arguments=my_dict_of_args)
+    >>> # Run template to return JSON output
+    >>> run_template(my_template_id, arguments=my_dict_of_args, JSONValue=True)
```

Comment:
I like the examples. Could you also show examples of what the output might look like? That would be helpful for me as a user trying to understand what this function does.